test/unit/FieldsetDate.test.js

```diff
@@ -0,0 +1,146 @@
+/* eslint-env mocha */
```

Comment:
Due to the using of `react-number-format`, the tests to simulate changes on input fields are not working. Haven't figure a way to do it so leave it commented out for now.